debian/control

```diff
@@ -0,0 +1,28 @@
+Source: bosh-agent
+Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
+XSBC-Original-Maintainer: Debian Go Packaging Team <team+pkg-go@tracker.debian.org>
```

Comment:
Nah, it wasn't in the Go team ever. This is just templating + update-maintainer that resulted in this.